templates/basic/test/import.mjs

```diff
@@ -0,0 +1,3 @@
+// import * as module from 'your-package-name';
+import * as module from '../dist/index.mjs';
+console.log(module);
```

Comment:
Test ESM import. If we generated this dynamically, the first line would work because of the package.json `"exports"` field: `import * as module from 'my-package-name'`